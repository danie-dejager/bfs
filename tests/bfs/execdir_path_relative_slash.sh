PATH="foo:$PATH" bfs_diff basic -execdir /bin/sh -c 'printf "%s\\n" "$@"' sh {} +
