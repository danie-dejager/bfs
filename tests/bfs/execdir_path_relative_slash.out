./a
./b
./bar
./bar
./basic
./baz
./c
./d
./e
./f
./foo
./foo
./foo
./g
./h
./i
./j
./k
./l
